### PR TITLE
views: decode base64.b64decode() as it returns bytes in py3

### PR DESCRIPTION
**Pull request by tanyunshi**
@fbochu

### CODE REVIEW

File: dokang/views.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def doc_upload(request):  # Route is not activated when dokang.uploaded_docs.dir
         token is None or
         request.authorization is None or
         request.authorization[0] != 'Basic' or
-        base64.b64decode(request.authorization[1]) != 'dokang:{0}'.format(token)
```
Comment (dzen) [inline dokang/views.py]: does it still works in python2 ?

Comment (tanyunshi) [inline dokang/views.py]: Apparemment oui
```
Python 2.7.9 (default, Mar  1 2015, 12:57:24) 

>>> x_str = str('a_secret')
>>> x_unicode = str('a_secret')
>>> x_str == x_unicode
True
>>> x_str.decode('utf-8') == x_unicode
True
>>> x_str.decode('utf-8') == x_str
True
```

Comment (tanyunshi) [inline dokang/views.py]: Apparemment oui

```
Python 2.7.9 (default, Mar  1 2015, 12:57:24)
>>> x_str = 'a secret'
>>> x_unicode = u'a secret'
>>> x_str == x_unicode
True
>>> x_str.decode('utf-8') == x_str
True
>>> x_str.decode('utf-8') == x_unicode
True

```

Comment (tanyunshi) [inline dokang/views.py]: PS. Dans py2,  `base64.b64decode` donne `str`.

Comment (dzen) [inline dokang/views.py]:
```>>> toto = u'é'
>>> toto.decode()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 0: ordinal not in range(128)
>>> toto.decode('utf-8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/encodings/utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 0: ordinal not in range(128)

>>> toto = 'é'
>>> toto.decode()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 0: ordinal not in range(128)

```

File: dokang/views.py [hunk 2]
```diff
@@ -147,11 +150,15 @@ def doc_upload(request):  # Route is not activated when dokang.uploaded_docs.dir
     doc_dir = settings['dokang.uploaded_docs.dir']
     token = settings.get('dokang.uploaded_docs.token')
 
+    http_auth = base64.b64decode(request.authorization[1])
+    if IS_PY3:
```
Comment (fbochu) [inline dokang/views.py]: `if isinstance(http_auth, bytes):` ?

Comment (dzen) [inline dokang/views.py]: c'est beaucoup plus clair avec la diff py2 / py3

**Comment by tanyunshi** [comment]
closed by 18843fd

**Pull request closed without merging**